packages/examples/source/src/adapter.ts

```diff
@@ -30,6 +30,6 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
   }
 }
 
-export const makeExecute: ExecuteFactory<Config> = (config) => {
-  return async (request) => execute(request, config || makeConfig())
+export const makeExecute = (config?: Config): Execute => {
```

Comment:
Similar to what we were talking about with `ExecuteWithConfig`, what is the goal of this change?

Reply:
I used that makeExecute in the Bob adapter and I think it was more flexible with the config input which I needed in for that adapter. I can revert it back if you think I should though.

Reply:
Yeah, we still want to maintain consistency across adapters with a shared type. Instead what we should do is change it at a source. 

In `packages/core/types/@chainlink/index.d.ts` you can see all of our repository level types.
You ran into trouble with `api` not being on Config, but the type requiring it right? The change would be to make `api` optional on the Config.

```
  export type Config = {
    apiKey?: string
    network?: string
    returnRejectedPromiseOnError?: boolean
    verbose?: boolean
    api?: RequestConfig
  }
```

and then to ensure this doesn't break anything by trying to compile (`yarn setup` from root) and lint again (`yarn lint` from root).

Then if this looks fine I would change the Bob adapter to use our shared types for consistency.

Reply:
Ok great, I will make this change. Thanks